README.md

```diff
@@ -59,6 +65,16 @@ $ rake lint
 
 Running specific tests is not currently supported.
```

Comment:
No, I just added the instructions to run one test *file*; the only way I know of to run one specific *test* with minitest is the `-n` option (`--name PATTERN Filter run on /pattern/ or string.`) which I find annoying. When I want to do this, I usually bring [`m`](https://rubygems.org/gems/m) into the picture, but I didn't want to add another dependency if you didn't want it :)